Controller/Config/Fast.php

```diff
@@ -103,7 +103,18 @@ public function execute()
         $quote = $this->checkoutSession->getQuote();
         $cartId = $this->getInvisibleCartFromCart($quote);
         $result = $this->resultJsonFactory->create();
+        $cartIsFast = true;
+        foreach ($quote ->getAllVisibleItems() as $cartItem) {
+            if ((int)$cartItem->getProduct()->getData('hide_fast_option') == 0) {
+                $cartIsFast = false;
+            }
```

Comment:
Is this logic correct? If `hide_fast_option` is 0, then doesn't that mean Fast is enabled for the product? Should we be checking if `hide_fast_option` is 1 instead?

Reply:
@corywebb , good question, i have copied this check logic from another existing function in Controller/Cart/Check.php that was used before on minicart page. Assumed functionality worked properly since was not a reported issue before. @jnodwell , could you please review this comment from Cory.

Reply:
regarding  Controller/Cart/Check.php - only reference to this file was from mini-cart, this call was removed and replaced with single call to `fast/config/fast`.
https://github.com/fast-af/magento2/pull/11#issuecomment-973125932